src/cache/cache_las.c

```diff
@@ -18,6 +18,8 @@
 	(WT_SESSION_IGNORE_CACHE_SIZE | WT_SESSION_READ_WONT_NEED |	\
 	WT_SESSION_NO_RECONCILE)
 
+#define	WT_LAS_FILE_MIN	(100 * WT_MEGABYTE)
```

Comment:
This should be defined in `include/cache.h` about line 57 with the other `WT_LAS_*` defines.

Reply:
ca144d1